### DISCUSSION TITLE
in near future can we use Electron as api with pyscript to make desktop apps

### DISCUSSION THREAD

**Discussion by engamir1** [pyscript/pyscript#1143] [General]
in near future can we use Electron as api with pyscript to make desktop apps

**Comment by FabioRosado** [comment]
I haven't tried using Electron with Pyscript yet, but in theory it should be possible. You may need to download the necessary files (pyodide and pyscript) and serve them locally to improve performance, you can have a look at the following repo that @tedpatrick  created which will allow you to download all the necessary files with a single command 😄 

https://github.com/tedpatrick/pyscript-local-runtime
